docs/index.md

```diff
@@ -1,3 +1,41 @@
-# Open Podcast API specification hub
+# Open Podcast API
 
-This site contains the specifications and documentation for the Open Podcast API.
+Welcome! The Open Podcast API is an initiative aiming to provide a feature-complete synchronisation API specification for podcast (web) apps and user-focussed servers.
+
+Our goal indirectly serving end-users: providing an easy way to sync subscriptions, listening progress, favourites, queu(s) and more between different apps and online services, and the ability to switch providers without hassle. Our goal for developers: providing reliable, open specifications which are decentralisation-ready and are easy to implement. To reach these goals we're deveveloping these specs in a collaborative way and sharing them in an [OpenAPI](https://spec.openapis.org/oas/latest) format.
+
+## Supported feature set & 'alpha' status
+
+Currently the specifications to synchronise [subscriptions](/docs/subscriptions) has been made available.
+
+We encourage all projects offering podcast listening and/or synchronisation functionality to adopt and implement defined specifications. However, please note that the **specifications are currently in 'alpha' state**: breaking changes might occur as initial implementation of these specs might reveal flaws in our theory.
+
+First next on the roadmap are specifications for authentication and basic episode status synchronisation.
```

Comment:
Maybe its better to Link to the Roadmap (eg tickets here on Github) so we don't need to change this section every time we finished something :D

Reply:
> For the sake of maintainability its always good to have as less moving parts as possible, eg the next roadmap items.

I agree in principle. I think, however, that we should be pragmatic and limit the overhead/effort. At this stage, updating the index page with each PR that adds new specs is small. Brainstorming about all the end-points/calls that we might want to have in future, creating issues for each of them and assigning them a date or priority to get a proper 'roadmap' would be more work.

Given the speed at which we move (which is perfectly fine) I don't expect we need any change for (at least) the rest of this year. But we can of course change at any time, if & when we feel like creating & maintaining a proper roadmap.